crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod coded;
```

Comment:
I would lift all files in coded one layer up (there is no non-coded settlement so creating this extra module in between doesn't add value IMO. Transaction isn't settlement specific.